droidlet/lowlevel/hello_robot/remote/remote_hello_robot.py

```diff
@@ -111,6 +114,17 @@ def get_base_state(self):
         s = self._robot.get_status()
         return (s["base"]["x"], s["base"]["y"], s["base"]["theta"])
 
+    def get_latest_lidar_scan(self):
+        return self._lidar.get_latest_scan()
+
+    def is_lidar_obstacle(self):
```

Comment:
what is this for? not used anywhere.

Reply:
It's not used, but I found it useful for debugging and inspecting robot state, such as from the ipython notebook. It seemed like something that was convenient to keep around.